puppet/modules/web/manifests/vhost/yum.pp

```diff
@@ -105,4 +105,19 @@
     ensure => link,
     target => '../nightly',
   }
+
+  file { "${yum_directory}/pulpcore":
+    ensure => directory,
+    owner  => $user,
+    group  => $user,
+    mode   => '0755',
+  }
+
+  file { "${yum_directory}/pulpcore/HEADER.html":
+    ensure  => file,
+    owner   => $user,
+    group   => $user,
+    mode    => '0644',
+    content => file('web/yum/pulpcore-HEADER.html'),
```

Comment:
the file in this commit is called `web/yum/pulp-HEADER.html`

Reply:
Not anymore ;)